data/i18n/pt-pt/OWNERS

```diff
@@ -0,0 +1,13 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+# Upstream (European Portuguese) localized strings.
+# Teams and members are visible at https://github.com/orgs/kubernetes/teams.
+
+reviewers:
+- sig-docs-pt-reviews
+
+approvers:
+- sig-docs-pt-pt-owners
```

Comment:
For now, I suggest:
```suggestion
- sig-docs-pt-owners
```
but eventually we may want to split the approvers for Portuguese into two separate teams.